tabbycat/privateurls/utils.py

```diff
@@ -24,6 +28,6 @@ def populate_url_keys(people, length=8, num_attempts=10):
     Person.objects.bulk_update(people, ['url_key'])
 
 
-def delete_url_keys(queryset):
+def delete_url_keys(queryset: 'QuerySet') -> None:
```

Comment:
URL keys are only on `Person`, so the type for `QuerySet` could be made more specific.